vm.go

```diff
@@ -151,8 +151,8 @@ func (vm *VM) EvaluateSnippetStream(filename string, snippet string) (docs []str
 	return
 }
 
-// EvaluateSnippetStream evaluates a string containing Jsonnet code to an array.
-// The array is returned as an array of JSON strings.
+// EvaluateSnippetMulti evaluates a string containing Jsonnet code to key-value
+// pairs. The keys and values are both returned as JSON strings.
```

Comment:
"The keys and values are both returned as JSON strings." sounds like it returned an an array of pairs of pair of arrays of strings.  And it actually returns a map[string]string. And while values in this map are JSONs, keys are not.

Reply:
That was careless of me. Fixed.